packages/web-react/src/components/Pagination/UncontrolledPagination.tsx

```diff
@@ -45,7 +48,7 @@ const UncontrolledPagination = (props: SpiritUncontrolledPaginationProps): JSX.E
             href="#"
```

Comment:
??

Reply:
Maybe `role="button"` will be better here.

Reply:
I would just like to point out that I haven't changed anything here. This is in the main branch